apps/web/src/lang/fr.json

```diff
@@ -8136,20 +8136,182 @@
+    "description": "Descriptive text explaining a valid post secondary education."
+  },
+  "2je6Bi": {
+    "defaultMessage": "Expérience minimale ou formation équivalente",
+    "description": "Heading for education requirements section of the application review page."
+  },
+  "2lOey6": {
+    "defaultMessage": "Votre CV",
+    "description": "Heading for resume section of the application review page."
+  },
+  "4ii2WZ": {
+    "defaultMessage": "Révisez votre candidature et envoyez-la!",
+    "description": "Subtitle for the application review page."
+  },
+  "5A0a7w": {
+    "defaultMessage": "Modifiez la liste des questions de sélection",
+    "description": "Edit link text for screening questions section of the application review page."
+  },
+  "6QGFk9": {
+    "defaultMessage": "Modifiez la liste des compétences requises",
+    "description": "Edit link text for skill requirements section of the application review page."
+  },
+  "7vOa71": {
+    "defaultMessage": "Nous avons bien reçu votre candidature",
+    "description": "Success message after submission for the application review page."
+  },
+  "DXtgrn": {
+    "defaultMessage": "Révisez votre candidature et envoyez-la",
+    "description": "Link text for the application review page."
+  },
+  "A9Urm+": {
+    "defaultMessage": "Lorsque vous envoyez votre candidature, une copie de votre profil est automatiquement créée.",
+    "description": "List item note 1 for the application review page."
+  },
+  "GXmpsW": {
+    "defaultMessage": "Avant de poursuivre, il est important que vous compreniez les remarques suivantes concernant votre candidature :",
+    "description": "Starting message for the application review page."
+  },
+  "JxDtt+": {
+    "defaultMessage": "Révisez votre candidature",
+    "description": "Page title for the application review page."
+  },
+  "MZrE40": {
+    "defaultMessage": "Vous avez dit répondre aux <strong>exigences minimales d’expérience ou de formation (deux années d’études postsecondaires)</strong>  et posséder l’expérience professionnelle suivante :",
+    "description": "Message on education requirements card on the application review page."
+  },
+  "O83lYZ": {
+    "defaultMessage": "Cette section résume votre CV tel que le verront les responsables du recrutement pour ces postes.",
+    "description": "Blurb for resume section of the application review page."
+  },
+  "Sd8883": {
+    "defaultMessage": "Vous avez répondu aux questions suivantes :",
+    "description": "Message in screening questions section of the application review page."
+  },
+  "T8eOrg": {
+    "defaultMessage": "Votre nom complet",
+    "description": "Label displayed for signature input in sign and submit section of application review page."
+  },
+  "PDbqFR": {
+    "defaultMessage": "\"J’ai relu tout le contenu de ma candidature\"",
```

Comment:
```suggestion
    "defaultMessage": "«J’ai relu tout le contenu de ma candidature»",
```

Reply:
Done in [cleanup french strings](https://github.com/GCTC-NTGC/gc-digital-talent/pull/6668/commits/49e1e777e42041f282522d72d7886ae146162339)